org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -40,7 +40,7 @@ GithubOidcSageBionetworksIt:
 GithubOidcSageBionetworksRecover:
   Type: update-stacks
   DependsOn: GithubOidcSageBionetworks
-  Template: github-oidc-provider-access.j2
+  Template: github-oidc-provider-access.njk
```

Comment:
For my info... is this a new templating engine?

Reply:
yes, it's nunjucks as opposed to jinja